### PR TITLE
Show tooltip on Triggers tab when it is disabled

### LINKED ISSUE #4865: Have a pop up over the Trigger tab to tell the user that they need to add a trigger first
### Feature description

The Trigger tab is greyed out until there is a trigger present in the flow. This can be confusing as you might assume you would configure a trigger from scratch in this menu.

Can also be confused as a paid feature too (like the greyed out namespace tabs).

To resolve any confusion, add a little hover pop up to say "Add a Trigger in the Editor first"
tgiggers

### PR DESCRIPTION
**Pull request by mounikaperi**
### What changes are being made and why?

Closes https://github.com/kestra-io/kestra/issues/4865

Currently, In flows page, when there is no trigger added, triggers tab is in disabled state.
We would like to add a tooltip when hovered on the triggers tab to intimate user to add the trigger first in the editor.
This would improve readability and understandability on why triggers tab is disabled on the page

Smoke Test Results
TriggerPopUpLightTheme
TriggerPopUp

---

### How the changes have been QAed?

---

### CODE REVIEW

File: ui/src/translations/en.json [hunk 1]
```diff
@@ -230,6 +230,7 @@
     "automatic refresh": "Automatic refresh",
     "toggle periodic refresh each 10 seconds": "Toggle periodic refresh every 10 seconds",
     "trigger refresh": "Trigger refresh",
+    "add-trigger-in-editor": "Add a Trigger in the Editor first",
```
Comment (wrussell1999) [inline ui/src/translations/en.json]: I'd change this to be more generic

```suggestion
    "add-trigger-in-editor": "Add a Trigger to your Flow first",
```

**Review by wrussell1999 — APPROVED** [review]
Looks good on my end but will let @MilosPaunovic review it from a code perspective!

File: ui/src/translations/en.json [hunk 2]
```diff
@@ -230,6 +230,7 @@
     "automatic refresh": "Automatic refresh",
     "toggle periodic refresh each 10 seconds": "Toggle periodic refresh every 10 seconds",
     "trigger refresh": "Trigger refresh",
+    "add-trigger-in-editor": "Add a Trigger to your Flow first",
```
Comment (MilosPaunovic) [inline ui/src/translations/en.json:233]: Rename key to `no_flow_triggers` and make value sentence case.

File: ui/src/components/Tabs.vue [hunk 3]
```diff
@@ -9,11 +9,24 @@
...
+                    <el-tooltip v-if="tab.disabled && tab.props && tab.props.showTooltip" :content="$t('add-trigger-in-editor')" placement="top">
+                        <span><strong>{{ tab.title }}</strong></span>
+                    </el-tooltip>
+                    
+                    <span v-else>
                         {{ tab.title }}
-                        <el-badge :type="tab.count > 0 ? 'danger' : 'primary'" :value="tab.count" v-if="tab.count !== undefined" />
-                    </enterprise-tooltip>
+                    </span>
+                    <el-badge
```
Comment (MilosPaunovic) [inline ui/src/components/Tabs.vue]: Please revert this formatting change is it is only formatting, to keep history as is.

File: ui/src/components/Tabs.vue [hunk 4]
```diff
@@ -9,11 +9,24 @@
...
             <template #label>
-                <component :is="embedActiveTab || tab.disabled || tab.locked ? 'a' : 'router-link'" @click="embeddedTabChange(tab)" :to="embedActiveTab ? undefined : to(tab)" :data-test-id="tab.name">
-                    <enterprise-tooltip :disabled="tab.locked" :term="tab.name" content="tabs">
+                <component 
+                    :is="embedActiveTab || tab.disabled || tab.locked ? 'a' : 'router-link'" 
+                    @click="embeddedTabChange(tab)" 
+                    :to="embedActiveTab ? undefined : to(tab)" 
+                    :data-test-id="tab.name"
+                >
+                    <el-tooltip v-if="tab.disabled && tab.props && tab.props.showTooltip" :content="$t('add-trigger-in-editor')" placement="top">
```
Comment (MilosPaunovic) [inline ui/src/components/Tabs.vue:13]: We can not remove the `enterprise-tooltip`, as we're showing it on tabs which are enterprise only, see the last one, `Audit logs`. So get that back, and use your logic as an extra.

And also, revert all unnecessary formatting changes to keep code ownership as intact as possible.

**Review by MilosPaunovic — CHANGES_REQUESTED** [review]

File: ui/src/translations/en.json (hunk 2 above)
Comment (wrussell1999) [reply ui/src/translations/en.json:233]: Could this wording be confused with Flow Triggers?

Comment (MilosPaunovic) [reply ui/src/translations/en.json:233]: Aren't these flow triggers? Or am I mixing something (which can very easy be the case)?

Bare in mind that this is only internal key to reference ćhe translation string value.

Comment (wrussell1999) [reply ui/src/translations/en.json:233]: This is for any triggers. But true, it's not something the user will see!

Comment (MilosPaunovic) [reply ui/src/translations/en.json:233]: Fair point. I'm absolutely okay with leaving it as is then.

**Comment by mounikaperi** [comment]
@MilosPaunovic can you please check now
I have added enterprise tooltip again

**Review by MilosPaunovic — APPROVED** [review]
LGTM! 🚀
